src/backend/metal/src/device.rs

```diff
@@ -613,13 +613,25 @@ impl hal::Device<Backend> for Device {
             .map(|attachment| attachment.borrow().clone())
             .collect::<Vec<_>>();
         let mut color_channels = Vec::new();
+        let mut depth_stencil_aspects = format::Aspects::empty();
+
         for attachment in &attachments {
             let is_depth = match attachment.format {
                 Some(f) => f.is_depth(),
```

Comment:
Probably better to check `|| f.is_stencil()` too (i.e. `is_depth_or_stencil` for `S8Uint` we don't have that in Metal yet though)

Reply:
yep